.test-infra/jenkins/committers_list_generator/templates/Committers.groovy.template

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * That's an automatically generated file. It contains the list of Committers' GitHub usernames. It's
```

Comment:
Is this an auto generated file? Or is this the template for the auto generated file?

Reply:
This is a template. The comment is here in order to make it clear (while reading the actual output file) that it's generated. Here, I thought this is rather clear with the `.template` suffix and `templates` dir, but I may be biased ;) Any ideas on how to reduce ambiguity?